src/pattern.class.ts

```diff
@@ -144,7 +144,7 @@ export class Pattern {
    * @param {Array} [propertiesToInclude] Any properties that you might want to additionally include in the output
    * @return {object} Object representation of a pattern instance
    */
-  toObject(propertiesToInclude: (keyof this)[]) {
+  toObject(propertiesToInclude?: (keyof this)[]) {
```

Comment:
pretty much sure will have to go back to Record<string, any> for this, but for now let's try